ui/webui/src/components/storage/StorageConfiguration.jsx

```diff
@@ -109,42 +47,40 @@ function AvailabilityState (available = true, reason = null, hint = null, shortH
 }
 
 // TODO total size check could go also to disk selection screen
-const checkEraseAll = async (selectedDisks, requiredSize) => {
-    const size = await getDiskTotalSpace({ diskNames: selectedDisks }).catch(console.error);
+const checkEraseAll = async ({ requiredSize, diskTotalSpace }) => {
```

Comment:
Is this still async?

Reply:
Good point - looks like no.

Reply:
This commit is split out in https://github.com/rhinstaller/anaconda/pull/4938

WIll fix it there.